.github/workflows/golangci-lint.yaml

```diff
@@ -0,0 +1,30 @@
+name: Lint & Annotate
```

Comment:
```suggestion
name: Lint & Annotate for Golang
```

Or something like that.  Basically we have multiple languages so we should distinguish in the name that this is for the golang code.

Reply:
Maybe "Lint & Annotate (Golang)"

Reply:
done